components/modules/groups/GroupMemberInfoCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import ListItemInfoCard from '@/components/ui/card/ListItemInfoCard';
+import theme from '@/constants/Colors';
+import { DiamondIcon } from '@/constants/Icon';
+import useProfilePicture from '@/hooks/attachment/UseProfilePicture';
+import { GroupMemberDetails } from '@/hooks/userdetails/UseGroupMembersDetails';
+import { getName } from '@/util/GetName';
+
+interface GroupMemberCardProps {
+  member: GroupMemberDetails;
+  isOwner: boolean;
+}
+
+const GroupMemberInfoCard: React.FC<GroupMemberCardProps> = ({ member, isOwner }) => {
+  const { data: profilePicture } = useProfilePicture(member.id, member.attachmentId);
+
+  return (
+    <ListItemInfoCard
+      image={!profilePicture ? { uri: '' } : profilePicture}
+      title={getName(member.firstName, member.lastName)}
+      details={member.username}
+      iconProps={
+        isOwner
```

Comment:
could we replace that with && without using undefined

Reply:
can't do it inside properties

Reply:
two options

```tsx
const iconProps = isOwner
? {
icon: <DiamondIcon />,
color: theme.primary.base,
}
: {};

const iconProps2 = {
...(isOwner && {
icon: <DiamondIcon />,
color: theme.primary.base,
}),
};
```

Reply:
both don't work (empty object !== undefined)

Reply:
but it works:
`  const iconProps = isOwner
    ? {
        icon: <DiamondIcon />,
        color: theme.primary.base,
      }
    : undefined;`